### ISSUE TITLE
Consider adding a title/label to the video element in the player for accessibility

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/video.js#2770]
Even if the HTML5 tech video player does not have controls, it seems to receive keyboard focus and be announced as simply 'region' by a screen reader. Whether it makes sense for video.js to add a label to the <video> element, or to remove it from the keyboard focus (tab) order, or whether this should be left to the HTML author needs some investigation, once screen reader compatibility of the rest of the video.js controls has been improved.

**Comment by gkatsev** [comment]
Thanks, we've considered it but decided to go with making [a plugin](https://github.com/brightcove/videojs-dock) for it previously before pulling it into core. This gives it even more reason to get pulled into core.

**Comment by OwenEdwards** [comment]
Closed by fa27cc7dbf601239999b0f99ca18318623da863f and optionally the [videojs dock plugin](https://github.com/brightcove/videojs-dock) which @gkatsev suggested.

**Comment by gkatsev** [comment]
@OwenEdwards Do we want to integrate the dock plugin into videojs? If so, we could keep this issue open until we do that.
